README.md

```diff
@@ -16,22 +16,22 @@
 Install module and required peer dependencies.
 
 ```
-$ npm install mali grpc @grpc/proto-loader
+$ npm install mali @grpc/grpc-js @grpc/proto-loader
 ```
 
 ## Example
 
 ```js
-const Mali = require('mali')
+import Mali from 'mali'
```

Comment:
minor nit, but I am not sure why are we using ESM in examples. My understanding it's not supported in Node.js natively w/o using special runtime flags and/or file extensions? Maybe things have changed?